src/scss/theme/_book.scss

```diff
@@ -3,7 +3,7 @@
     overflow: hidden;
     perspective: 350px;
     height: 470px;
-    background-color: #15312C;
+    background-color: #15312c;
```

Comment:
These MUST be SCSS variables defined inside `src/scss/theme/_variables.scss`
Apply to the rest please.

Reply:
Will do for all on code clean up later, these changes are not from this PR.